packages/clerk-js/src/ui/elements/InputGroup.tsx

```diff
@@ -41,18 +40,28 @@ export const InputGroup = forwardRef<
       hasError={rest.hasError}
       sx={theme => ({
         position: 'relative',
-        borderRadius: theme.radii.$md,
         zIndex: 1,
-        border: theme.borders.$normal,
-        borderColor: theme.colors.$blackAlpha300, // we use this value in the Input primitive
+        ...common.borderVariants(theme).normal,
       })}
     >
-      {groupPrefix && <Text sx={textProps}>{groupPrefix}</Text>}
+      {groupPrefix && (
+        <Text
+          colorScheme='neutral'
+          sx={textProps}
+        >
+          {groupPrefix}
+        </Text>
+      )}
       <Input
         maxLength={25}
         sx={[
           {
-            borderColor: 'transparent',
+            boxShadow: 'none',
+            border: 'none',
+            ':hover:not(:focus)': {
+              border: 'none',
+              boxShadow: 'none',
+            },
```

Comment:
Is is better if we add a new variant for this case?

Reply:
I'm not sure, I think that's only in 2 places.